### ISSUE TITLE
The validation loss and accuracy

### ISSUE THREAD

**Issue by lgbwust** [fchollet/keras#161]
I use the framework to train a RGB image dataset, the train loss is decreasing and the accuracy is increasing, but the validation loss is decreasing at first, then the loss is increasing, what's the reason? Thank you very much!

**Comment by vzhong** [comment]
Hi @lgbwust , this is not exactly a Keras related issue. You are likely overfitting on your training set. The [user forum] would probably be a better place for this.

**Comment by cjmcmurtrie** [comment]
https://en.wikipedia.org/wiki/Overfitting
